src/chapters/setup-intro-data-types/studio.rst

```diff
@@ -1,2 +1,68 @@
 Studio: Data Types
 ===================
+
+Get cosy with Java syntax by writing a console program that calculates the 
+area of a circle based on input from the user.
+
+Creating your class
+-------------------
+
+Since you’re still new to Java and IntelliJ, we’ll provide some extra
+direction the first few times we go to write code.
+
+Create a new package named ``org.launchcode.java.studios`` by
+right-clicking (or ctrl-clicking for some Mac users) on the ``src`` directory 
+in ``java-web-dev-exercises`` and selecting *New > Package*. Be sure to enter 
+the full name, or your package won’t be created in the correct location.
+
+Create your program/class in the ``java-web-dev-exercises`` project within the
+package ``org.launchcode.java.studios`` by right-clicking/ctrl-clicking on the
+``studios`` package/folder and selecting *New > Java Class*. Enter the
+name ``Area``. Select the option to add the file to Git when the window appears.
+
+Your task
+---------
+
+Write a program/class ``Area`` that prompts the user for a number, and
+then calculate the area of a circle with that radius and print the
+result.
+
+.. admonition:: Tip
+   
+   Recall that the area of a circle is ``A = pi * r * r`` where ``pi`` is
+   3.14 and ``r`` is the radius.
+
+
+Here’s an example of how your program should work:
+
+.. sourcecode:: java
+
+   Enter a radius: 2.5
+   The area of a circle of radius 2.5 is: 19.625
+
+Some questions to ask yourself: 
+
+#. What data type should the radius be? 
+#. What is the best way to get user input into a variable ``radius`` of
```

Comment:
In the Tip admonition, ``r`` was used for the radius.  Change one or the other to make the names consistent.

Reply:
the tip is just a reminder of what the formula is, not a code sample. keeping them different will help stimulate debugging if a student copy/pastes

Reply:
Ah.  Clever.